vertx-core/src/main/java/io/vertx/core/http/FileSender.java

```diff
@@ -0,0 +1,51 @@
+package io.vertx.core.http;
+
+import io.vertx.codegen.annotations.GenIgnore;
+import io.vertx.core.Future;
+
+import java.nio.channels.Channel;
+
+public interface FileSender<T extends Channel> {
```

Comment:
overall I don't get why we need to introduce this new interface instead of having methods on HttpServerResponse to send files like we do have for others